tests/errors/test_logical_compare_01.py

```diff
@@ -0,0 +1,5 @@
+def f():
+    print([1, 2, 3] or [1, 2, 3, 4])
+
+
+f()
```

Comment:
I think we do not need this test. It is redundant at the moment. Both the operands of the operation are of the same type, so we do not expect any SemanticError here. It is a different case that we do not support lists in `selective or` operation for which we can open an issue.

Reply:
Please ensure that you also remove the reference outputs associated with this test.